proxy/envoy/config.go

```diff
@@ -98,7 +101,7 @@ func Generate(instances []*model.ServiceInstance, services []*model.Service, mes
 		*/
 		Listeners: listeners,
 		Admin: Admin{
-			AccessLogPath: "/dev/stdout",
+			AccessLogPath: "/dev/null",
```

Comment:
No, it should be /dev/stdout (although, I think Mixer is going to monitor proxies themselves)